lib/resources/nginx_conf.rb

```diff
@@ -82,10 +82,13 @@ def resolve_references(data, rel_path)
       # Any call to `include` gets its data read, parsed, and merged back
       # into the current data structure
       if data.key?('include')
-        data.delete('include').flatten
-            .map { |x| File.expand_path(x, rel_path) }
-            .map { |path| parse_nginx(path) }
-            .map { |e| data.merge!(e) }
+        include_paths = data.delete('include').flatten.map do |path|
+          paths = inspec.command("ls #{File.expand_path(path, rel_path)}").stdout.split
```

Comment:
Hi All,

Does anyone have any thoughts on the issue around using the command resource in the code? I was googling around and for the doing a Dir.glob() remotely was accomplished using the Net:SSH.command option given that the ruby Dir.glob() executes locally. Thinking along those lines, I was wondering if we could use the train connection object in the inspec runtime in a similar way and then use the train.os and or train.family to then set the "DIR" command that is sent - i.e. `ls` for unixies and `dir` for windows. I could be totally off base here.